src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -208,6 +208,11 @@ const generateTaskRole = (taskRole, labels, config) => {
       count: ports[port],
     };
   }
+  // get shared memory size
+  let shmMB = 512;
```

Comment:
Could we on demand mount, don't set default value?

Reply:
even if we don't set the `shm-size`, there's also a default value in Docker (`shm-size=64m`)
just enlarge the default value here in case user don't know to set it